terraform/ingest/index.js

```diff
@@ -121,7 +121,9 @@ function processFiles () {
     // YYYYMMDD format
```

Comment:
Can you update this comment to indicate the correct format (output of `toISOString`)?

Reply:
Is `YYYY-MM-DD` sufficient or something like`YYYY-MM-DDThh:mm:ss.sssZ` to be specific?

Reply:
`YYYY-MM-DDThh:mm:ss.sssZ`, doesn't hurt to be specific.